src/dash-adapter.js

```diff
@@ -493,6 +495,7 @@ export default class DashAdapter extends BaseMediaSourceAdapter {
    */
   selectTextTrack(textTrack: TextTrack): void {
     if (this._shaka && (textTrack instanceof TextTrack) && !textTrack.active && (textTrack.kind === 'subtitles' || textTrack.kind === 'captions')) {
+      this._shaka.setTextTrackVisibility(this._config.textTrackVisibile);
```

Comment:
better to set it on `init` as before. no?

Reply:
we need it here, because for when calling hideTextTracks, e.g. setting them `OFF`, we call `setTextTrackVisibility(false)` so we need to always toggle it back.
There's no option in Shaka API to set subtitle OFF, just setting the display mode, so we have to call it when setting text track display of `selectTextTrack`